### PR TITLE
Feat/auth

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change logs

- Added firebase in the app
- Added Auth in the App
- Added navbar
- added header and footer

## Preview
https://deploy-preview-1--ripple-social.netlify.app/

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+				<div className="brand-info flex-row justify-content-center align-center flex-gap-1 m-5">
+					{location.pathname !== "/auth" && (
+						<section>
+							<i
+								className="fas fa-bars header-nav-icon"
+								onClick={handleShowNavbar}
+							></i>
+						</section>
+					)}
+					<Link to={"/"} className="no-link header-brand">
```
Comment (vidhi-thakur) [inline src/components/Header/Header.jsx:26]: `to={"/"}`
you can remove curly braces here

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+						<li className="rui-drawer-content">
+							<NavLink to={"/profile"} className={getActiveClass}>
+								<div className="rui-drawer-links">
+									<span className="rui-drawer-content--text">
+										{authState.token ? "Profile" : "SignIn"}
+									</span>
+								</div>
+							</NavLink>
+						</li>
+					</ul>
```
Comment (vidhi-thakur) [inline src/components/NavBar/NavBar.jsx:64]: just a suggestion: the code inside <li> tags is repetitive, you can create an array of object here and map over the array

File: src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+}
+
+.rui-main--heading-container, .rui-main--sub-heading {
+  padding: var(--p-0);
+  margin: var(--m-0);
+  border-bottom: var(--b-0);
+}
+
+.focused > .textbox-label {
+	background: var(--secondary-background-color);
```
Comment (vidhi-thakur) [inline src/css/index.css:73]: can update the formatting of this file in next iteration

File: src/css/index.css [hunk 4]
```diff
@@ -0,0 +1,471 @@
...
+	display: block;
+}
+
+.vertical-tabs-content {
+  display: block;
+}
+
+.nav-container {
+  padding: var(--p-0);
+	z-index: var(--max-z-index);
```
Comment (vidhi-thakur) [inline src/css/index.css:109]: good work with variable and rems

**Review by vidhi-thakur — APPROVED** [review]
code looks good to merge

**Review by shashikantmaurya0007 — APPROVED** [review]
Vidhi has already pointed out the changes can be made.rest looks good to be merged

**Pull request merged**